libs/src/api/serviceProvider.ts

```diff
@@ -1,8 +1,10 @@
-const { sampleSize } = require('lodash')
+import { sampleSize } from 'lodash'
```

Comment:
A pattern ive been following is migrating the files to CapCase to accurately represent that this is a class file. what do you think about this?

Reply:
I like it!